schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/convert/fromJsonOpenDataSoft.xsl

```diff
@@ -135,6 +135,16 @@
             <cit:date>
               <gco:DateTime><xsl:value-of select="(metas/modified|dataset/metas/default/metadata_processed)[1]"/></gco:DateTime>
             </cit:date>
+            <cit:dateType>
+              <cit:CI_DateTypeCode codeList="codeListLocation#CI_DateTypeCode" codeListValue="publication"/>
+            </cit:dateType>
+          </cit:CI_Date>
+        </mdb:dateInfo>
+        <mdb:dateInfo>
+          <cit:CI_Date>
+            <cit:date>
+              <gco:DateTime><xsl:value-of select="metas/metadata_processed"/></gco:DateTime>
```

Comment:
As this XSLT conversion is providing mapping for V1 and V2 of ODS API, it would be kind to handle both mapping or at least to not create empty elements when the mapping is only improved for 1 version of the ODS API.